ci/docker/python-wheel-manylinux.dockerfile

```diff
@@ -25,7 +25,10 @@ ARG manylinux
 ENV MANYLINUX_VERSION=${manylinux}
 
 # Install basic dependencies
-RUN yum install -y git flex curl autoconf zip perl-IPC-Cmd wget 
+RUN yum install -y git flex curl autoconf zip perl-IPC-Cmd wget
+
+ENV CPYTHON_VERSION=cp38
+ENV PATH=/opt/python/${CPYTHON_VERSION}-${CPYTHON_VERSION}/bin:${PATH}
```

Comment:
I am not entirely sure why is this required and how does affect all the wheels we are generating for other Python versions. Could you share more details on why is this required and maybe add a comment?

Reply:
@raulcd See the previous discussion around this change here: https://github.com/apache/arrow/pull/33805#discussion_r1083333041.

As I understand it, programs in the Dockerfile require a system python installed (ninja and vcpkg). The manylinux2014 Dockerfile has python 2.7.5 while manylinux_2_28 has no system python installed. For this reason, we add the following to the PATH.

Reply:
I will add a comment.